tensilelite/Tensile/TensileCreateLibrary.py

```diff
@@ -361,65 +361,6 @@ def buildSourceCodeObjectFiles(CxxCompiler, kernelFiles, outputPath):
 
     return itertools.chain.from_iterable(coFiles)
 
-################################################################################
-def prepAsm(kernelWriterAssembly):
```

Comment:
Is this an unused function?

Reply:
That function generates the asm-new.sh script that is intended to be replaced by the Makefile added in this PR.